project/.travis.yml.twig

```diff
@@ -50,15 +50,15 @@ matrix:
 
 before_install:
   - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then mv "$HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini" /tmp; fi;
-  - composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN
-  - phpunit --self-upgrade
-  - sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json
-  - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+  - if [ "$TARGET" = 'test' ]; then composer config --quiet --global github-oauth.github.com $GITHUB_OAUTH_TOKEN; fi;
+  - if [ "$TARGET" = 'test' ]; phpunit --self-upgrade; fi;
+  - if [ "$TARGET" = 'test' ]; then sed --in-place "s/\"dev-master\":/\"dev-${TRAVIS_COMMIT}\":/" composer.json; fi;
+  - if [ "$SYMFONY_VERSION" != "" -a "$TARGET" = 'test' ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
 
 install:
-  - travis_wait composer global update --prefer-dist --no-interaction
-  - travis_wait composer update --prefer-dist --no-interaction --prefer-stable $COMPOSER_FLAGS
-  - pip install -r {{ docs_path }}/requirements.txt --user `whoami`
+  - if [ "$TARGET" = 'test' ]; then travis_wait composer global update --prefer-dist --no-interaction; fi;
```

Comment:
Since you removed phpunit, nothing is globally required. You can remove this line.